index.html

```diff
@@ -19,28 +19,167 @@ <h4>Yunus Ali</h4>
         </nav>
     </header>
     <main>
-        <section>
-        <article>
-            <h1>I’m Yunus
-                Glad to see you!</h1>
-            <p>
-                I’m a software developer!
-                I can help you build a product ,
-                feature or website Look through
-                some of my work and experience!
-                If you like what you see and have
-                a project you need coded, don’t hestiate
-                to contact me.
-            </p>
-        </article>
-        <h4>Lets connect</h4>
-        <ul>
-            <li><a href="#"><i class="bi bi-twitter"></i></a></li>
-            <li><a href="#"><i class="bi bi-linkedin"></i></a></li>
-            <li><a href="#"><i class="bi bi-medium"></i></a></li>
-            <li><a href="#"><i class="bi bi-github"></i></a></li>
-            <li><a href="#"><i class="bi bi-slack"></i></a></li>
-        </ul>
+        <section class="main-section">
+            <section class="headline">
+            <article>
+                <h1>I’m Yunus
+                    Glad to see you!</h1>
+                <p>
+                    I’m a software developer!
+                    I can help you build a product ,
+                    feature or website Look through
+                    some of my work and experience!
+                    If you like what you see and have
+                    a project you need coded, don’t hestiate
+                    to contact me.
+                </p>
+                <section class="connect">
+                    <h4>Lets connect</h4>
+                    <ul>
+                            <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+                            <li><a href="#"><i class="bi bi-linkedin"></i></a></li>
+                            <li><a href="#"><i class="bi bi-medium"></i></a></li>
+                            <li><a href="#"><i class="bi bi-github"></i></a></li>
+                            <li><a href="#"><i class="bi bi-slack"></i></a></li>
+                    </ul>
+                </section>
+            </article>
+            </section>
+
+            <section class="work">
+                    <div class="container">
```

Comment:
- [ ] kindly fix the padding/gap between your headline section and work section.
your design
![Screenshot (66)_LI](https://user-images.githubusercontent.com/65298048/170661954-f020e6be-53d5-4894-9026-15ee08439cb1.jpg)
figma design
![Screenshot (67)_LI](https://user-images.githubusercontent.com/65298048/170662002-d5051020-329f-4574-bd53-2ea86ae53449.jpg)

Reply:
the gab is 130 on figma mine is the same now